### PR TITLE
py sol for empty string

### PR DESCRIPTION
**Pull request by freakin23**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [ ] I have linked this PR to any issues that it closes.

### CODE REVIEW

**Comment by freakin23** [comment]
AC: https://cses.fi/paste/89bec1769095d49da053fe/

File: solutions/gold/cses-1080.mdx [hunk 1]
```diff
@@ -76,5 +79,50 @@ int main() {
...
+```py
+N = 500
+MOD = 10**9 + 7
+
+
+def add(a: int, b: int) -> int:
+	a += b
+	if a >= MOD:
+		a -= MOD
+	return a
```
Comment (SansPapyrus683) [inline solutions/gold/cses-1080.mdx:93]: you can just do `a + b % MOD`

Comment (freakin23) [reply solutions/gold/cses-1080.mdx]: done

**Comment by freakin23** [comment]
AC: https://cses.fi/paste/7c3739eab3a47a52a0741a/

**Review by SansPapyrus683 — APPROVED** [review]

**Pull request merged**